### PR TITLE
{devel} [dummy] GCC-5.1 (REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/618d63833157e74a7d1e for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/eb1fb72d2096ebc7c388 for a full test report.

File: easybuild/easyconfigs/g/GCC/GCC-5.1.0.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    'b7ff2d88cae7f8085bd5006096eed470',     # gmp-6.0.0a.tar.bz2
+    '181aa7bb0e452c409f2788a4a7f38476',     # mpfr-3.1.2.tar.gz
+    'd6a1d5f8ddea3abd2cc3e98f58352d26',     # mpc-1.0.3.tar.gz
+    'acd347243fca5609e3df37dba47fd0bb',     # isl-0.14.tar.bz2
+    '58aec98d15982f9744a043d2f1c5af82',     # mpfr-3.1.2-allpatches-20141204.patch
+]
+
+languages = ['c', 'c++', 'fortran']
+
+#withisl = True
```
Comment (boegel) [inline easybuild/easyconfigs/g/GCC/GCC-5.1.0.eb]: drop this?

**Comment by wpoely86** [comment]
Depends on hpcugent/easybuild-easyblocks#598

**Comment by wpoely86** [comment]
The last fail is unavoidable because it uses the old easyblock.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/4bebef66c21c1c413b86 for a full test report.

**Comment by fgeorgatos** [comment]
fyi. I've seen success with this, thanks for providing it:

```
$ time eb --from-pr 1553
== temporary log file in case of crash /tmp/eb-z4QsQA/easybuild-qVchGn.log
== processing EasyBuild easyconfig /tmp/eb-z4QsQA/files_pr1553/GCC-5.1.0.eb
== building and installing GCC/5.1.0...
== fetching files...
== creating build dir, resetting environment...
== unpacking...
== patching...
== preparing...
== configuring...
== building...
== testing...
== installing...
== taking care of extensions...
== packaging...
== postprocessing...
== sanity checking...
== cleaning up...
== creating module...
== COMPLETED: Installation ended successfully
== Results of the build can be found in the log file /opt/apps/HPCBIOS.20150417/software/GCC/5.1.0/easybuild/easybuild-GCC-5.1.0-20150423.035921.log
== Build succeeded for 1 out of 1
== temporary log file /tmp/eb-z4QsQA/easybuild-qVchGn.log has been removed.
== temporary directory /tmp/eb-z4QsQA has been removed.

real    41m2.219s
user    104m12.163s
sys     14m28.593s
```

**Comment by fgeorgatos** [comment]
btw./fyi. the above test was before your easyblock PR;

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/6c357e1c0edb02435c56 for a full test report.

**Comment by wpoely86** [comment]
You only need the new easyblock if you want ISL.

**Comment by boegel** [comment]
ISL is now enabled by default with the latest update, so the easyblock update is now also required

works like a charm on my end too, so going in!

Thanks a bunch @wpoely86!

**Comment by boegel** [comment]
Tested on top of https://github.com/hpcugent/easybuild-easyblocks/pull/598, looking great, so going in. Thanks @wpoely86!

**Comment by boegel** [comment]
Jenkins: test this please

**Pull request merged**